roles/uperf-bench/tasks/main.yml

```diff
@@ -1,109 +1,126 @@
 ---
-- name: Start Server(s)
-  k8s:
-    definition:
-      kind: Deployment
-      apiVersion: apps/v1
-      metadata:
-        name: '{{ meta.name }}-uperf-server-bench'
-        namespace: '{{ meta.namespace }}'
-      spec:
-        replicas: "{{uperf.pair}}"
-        selector:
-          matchLabels:
-            app: uperf-bench-server
-        template:
-          metadata:
-            labels:
+
+- block:
+
+  - name: Wait for old pods to destroy...
```

Comment:
There seems to be an issue with this logic, as it;s going into an inifinite loop and is not working.

Reply:
This was working against my OCP4 cloud? I will retest